### PR TITLE
Update CHANGELOG.md [5.1]

### PR DESCRIPTION
**Pull request by slavapestov**

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please test

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,9 +25,125 @@ CHANGELOG
 Swift 5.1
 ---------
 
+* [SE-0256][]:
```
Comment (benrimmington) [inline CHANGELOG.md]:
```suggestion
* [SE-0254][]:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7511,6 +7629,7 @@ Swift 1.0
 [SE-0228]: <https://github.com/apple/swift-evolution/blob/master/proposals/0228-fix-expressiblebystringinterpolation.md>
 [SE-0230]: <https://github.com/apple/swift-evolution/blob/master/proposals/0230-flatten-optional-try.md>
 [SE-0235]: <https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md>
+[SE-0242]: <https://github.com/apple/swift-evolution/blob/master/proposals/0242-default-values-memberwise.md>
 [SE-0245]: <https://github.com/apple/swift-evolution/blob/master/proposals/0245-array-uninitialized-initializer.md>
```
Comment (benrimmington) [inline CHANGELOG.md]:
```markdown
[SE-0252]: <https://github.com/apple/swift-evolution/blob/master/proposals/0252-keypath-dynamic-member-lookup.md>
[SE-0254]: <https://github.com/apple/swift-evolution/blob/master/proposals/0254-static-subscripts.md>
```

Comment (benrimmington) [reply CHANGELOG.md]: @slavapestov I can create pull requests for these changes.

**Pull request merged**